File: OneDrive 1.2 Feb 23 2024/Python/Alumno.py
class Alumno:
    def __init__(self, *args):
        if len(args)==0:
            self._noControl=""
            self._nombre=""
            self._calif1=0
            self._calif2=0
            self._calif3=0
        elif len(args)==5:
            self._noControl=args[0]
            self._nombre=args[1]
            self._calif1=args[2]
            self._calif2=args[3]
            self._calif3=args[4]           
        pass


    def Leer(self):
        print ("Leer")

    def RetNoControl(self):
        print ("Num Control")
        
    
    def RetNombre(self):
        print ("Nombre")
        pass

    def RetCalif(self):
        print ("Calificacion")
        

    def setNoControl(self, nc):
        self._noControl=nc

    def getNoControl(self):
        return self._noControl

    def setNombre(self, no):
        self._nombre=no
    
    def getNombre(self):
        return self._nombre
   
    def setCalif1(self, c1):
        self._calif1=c1
    
    def getCalif1(self):
        return self._calif1

    def setCalif2(self, c2):
        self._calif2=c2
    
    def getCalif2(self):
        return self._calif2

    def setCalif3(self, c3):
        self._calif3=c3
    
    def getCalif3(self):
        return self._calif3

alumno1 = Alumno()
alumno2 = Alumno("01230523", "Lizbeth", 89, 88, 90)

print(alumno1.getNombre())
print(alumno1.getNoControl())
print(alumno1.getCalif1())
print(alumno1.getCalif2())
print(alumno1.getCalif3())
print("Alumno 2")
print(alumno2.getNombre())
print(alumno2.getNoControl())
print(alumno2.getCalif1())
print(alumno2.getCalif2())
print(alumno2.getCalif3())

File: OneDrive 1.2 Feb 23 2024/C#/Alumno.cs
using System;

public class Alumno{
    //Atributos
    private String _noControl;
    private String _nombre;
    private int _calif1;
    private int _calif2;
    private int _calif3;
    
    //Constructores
    public Alumno(){
        this._noControl="";
        this._nombre="";
        this._calif1=0;
        this._calif2=0;
        this._calif3=0;
    }
    public Alumno(String nc, String no, int c1, int c2, int c3){
        this._noControl=nc;
        this._nombre = no;
        this._calif1=c1;
        this._calif2=c2;
        this._calif3=c3;
    }

    //Métodos setter y getter
    public void setNoControl(String nc){
        this._noControl = nc;
    }

    public String getNoControl(){
        return this._noControl;
    }

    public void setNombre(String no){
        this._nombre=no;
    }

    public String getNombre(){
        return this._nombre;
    }

    public void setCalif1(int c1){
        this._calif1=c1;
    }

    public int getCalif1(){
        return this._calif1;
    }

    public void setCalif2(int c2){
        this._calif2=c2;
    }

    public int getCalif2(){
        return this._calif2;
    }

    public void setCalif3(int c3){
        this._calif3=c3;
    }

    public int getCalif3(){
        return this._calif3;
    }

    //Métodos
    public String RetControl(){
        return "OP12344";
    }

    public String RetNombre(){
        return "Liliana";
    }

    public void RetCalif(){

    }



}